spec/abstract-jobs.html

```diff
@@ -100,6 +100,33 @@ <h1>Objectives</h1>
       </p>
     </emu-clause>
 
+    <emu-clause id="sec-liveness">
+      <h1>Liveness</h1>
+
+      An object _obj_ is considered <dfn>live</dfn> if the following conditions
```

Comment:
Wouldn't an object be considered **not** live in these conditions?

Reply:
Good catch, thanks, will need to fix this before landing.

Reply:
Fixed